### PR TITLE
tests(coverage): add more tests to reach 100%

### PR DESCRIPTION
**Pull request by bangn**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -69,6 +69,7 @@
     "clean": "rm -rf node_modules package-lock.json",
     "contributors": "(git-authors-cli && finepack && git add package.json && git commit -m 'build: contributors' --no-verify) || true",
     "coverage": "jest --coverage --coverageReporters=text-lcov | coveralls",
+    "coverage:dev": "jest --coverage",
```
Comment (kutyel) [inline package.json]: This is not needed, our `npm test` is already this command 😉

**Review by kutyel — CHANGES_REQUESTED** [review]
Looks great! Remove the extra npm script and this is good to be merged!

File: package.json (hunk 1 above)
Comment (bangn) [reply package.json]: Ah ha, I overlooked it :)

Comment (bangn) [reply package.json]: Fixed 2077b55

**Review by kutyel — APPROVED** [review]
Awesome, thanks! 🚀

**Pull request merged**